packages/unity-interface/BrowserInterface.ts

```diff
@@ -664,6 +666,11 @@ export class BrowserInterface {
     getUnityInstance().SendHeaders(data.url, headers)
   }
 
+  public async PublishSceneState(data: PublishPayload) {
+    const result = await deployScene(data)
+    getUnityInstance().SendPublishSceneResult(result)
+  }
```

Comment:
Prevent creation of async function in BrowserInterface, nobody catches the errors

```suggestion
  public PublishSceneState(data: PublishPayload) {
    deployScene(data).then(result => {
      getUnityInstance().SendPublishSceneResult(result)
    }).catch(error => defaultLogger.error(error))
  }
```